bridgerelayer/chains/ethereum/router.go

```diff
@@ -124,16 +126,16 @@ func (er Router) Submit(appID []byte, data []byte) error {
 	}
 
 	// Calculate the method ID of our function using crypto.sha3
-	submitFnSignature := []byte("submit(bytes)")
+	submitFnSignature := []byte("submit(data,sig)")
```

Comment:
We still need to align this method signature with the new contract design in https://github.com/Snowfork/polkadot-ethereum/pull/57